### DISCUSSION TITLE
Change baseurl by environment variable file

### DISCUSSION THREAD

**Discussion by hanwo** [acacode/swagger-typescript-api#799] [Q&A]
I'm using multiple environment variable files (ex: local,dev...)
Is it possible to dynamically change the baseurl per environment variable files?

```
"swagger-typescript-api": "^13.0.8"
"typescript": "5.3.3"
"axios": "^1.7.2",
```
and i use script
> swagger-typescript-api -p {file} -r -o {dir} --modular -d --extract-request-body --extract-response-body --extract-response-error --axios

**Comment by gpiatek** [comment]
You can pass `baseUrl` property when creating API instance
```
import { Api } from './generated-api';

export const myApiInstance = new Api({
  baseURL: '/MY_BASE_URL',
});
```
